### PR TITLE
BigAnimal: quick sign-in URL

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

https://enterprisedb.atlassian.net/browse/UPM-10159

Added a section in the IdP parent topic.

### CODE REVIEW

**Review by edcrewe — APPROVED** [review]
Hi Dee Dee.
Thanks for sorting out the appropriate help pages for each provider.
That all look goods to me.

File: product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx [hunk 1]
```diff
@@ -60,3 +60,7 @@ description: "Using AWS SSO as your identity provider for your Azure account"
    1. Select **Test Connection**.
    1. Enter the AWS user credentials that you granted access. 
    1. If the Test Connections is successful, then select **Sign in to BigAnimal** to complete the setup process in the BigAnimal portal.
+
+## Create a tile
+
+You can optionally use the quick sign-in URL provided on the **Settings > Identity Provider** page of the BigAnimal portal to create a tile for your users to access for the BigAnimal application. See the [Enable single sign-on access to your AWS applications](https://docs.aws.amazon.com/singlesignon/latest/userguide/use-case-app-admin.html) topic in the AWS documentation for details.
```
Comment (xugy99) [inline product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx:66]: reading this linked doc still makes me confused, I am still not clear on how to create a tile in AWS SSO by leveraging the quick-signin-URL. This linked doc shows to very vague and I am not even sure if it is talking about the same thing

Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx:66]: If that is the case, I suggest we remove the section and possibly create a KB article. @jasonparton, what are your thoughts?

Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx]: I removed the sections in the IdP-specific topics.

**Pull request merged**